test/helpers/createLazyTestEnv.js

```diff
@@ -77,26 +77,40 @@ module.exports = (globalTimeout = 2000, nameSuffix = "") => {
 			stack: block.asyncError.stack
 		};
 	};
+	function _it(...args) {
+		numberOfTests++;
+		if (runTests >= numberOfTests) throw new Error("it called too late");
+		args[1] = createDisposableFn(args[1], true);
+		args[2] = args[2] || globalTimeout;
+		inSuite(() => {
+			// eslint-disable-next-line jest/no-disabled-tests
+			it(...args);
+			fixAsyncError(
+				currentDescribeBlock.tests[currentDescribeBlock.tests.length - 1]
+			);
+		});
+	}
+	_it.failing = (...args) => {
```

Comment:
This seems to be some kind of custom test runner but `it.failing` should work with Jest: https://jestjs.io/docs/api#testfailingname-fn-timeout

Reply:
Hmm, I see `it.failing` isn't working in Node v10 CI because it uses an older version of Jest (`it.failing` was introduced in Jest 28)

Reply:
v10 should be dropped anyway. We should support v15 ++

Reply:
Incremental PR upgrading to Node v12 and Jest v28: https://github.com/webpack/webpack/pull/18574

Reply:
@evenstensberg According to @alexander-akait Node v10 is still supported https://github.com/webpack/webpack/pull/18574#discussion_r1679325664